### DISCUSSION TITLE
Unable to sign in Flash builder

### DISCUSSION THREAD

**Discussion by abhisheksb164** [airsdk/Adobe-Runtime-Support#1351] [Q&A]
On sign in Flash builder, it shows update required but OS and Browser are up-to-date. Anyone faced this situation.

Following is the message showing 
_Update required
Your browser or operating system is no longer supported. You may need to install the latest updates to your operating system.  Learn more_ 

I am using new machine with Windows 10

**Comment by FliplineStudios** [comment]
I've had the same message multiple times with other older Adobe CC software, from versions that have been installed on my machine for years.  They'll often suddenly require a sign-in and act like a trial version, and get caught in a loop of requiring a password or showing the update required popup.  Sometimes rebooting sorts it out on its own, other times it doesn't....

There are some suggestions on the adobe forums here:
https://community.adobe.com/t5/captivate-discussions/update-required-your-browser-or-operating-system-is-no-longer-supported/td-p/12193264

Adobe has some possible options here:
https://helpx.adobe.com/in/x-productkb/global/update-operating-system-and-browser.html

And some are saying that this other solution from Adobe may solve the problem (even though the solution relates specifically to the CC Packager):
https://helpx.adobe.com/enterprise/kb/ccp-windows-sign-in-issue.html

**Reply by abhisheksb164** [reply]
Thanks FliplineStudios, Sorry for this late reply. The issue was solved when I installed Adobe acrobat reader and its latest update.
May be Adobe's latest login changes are included with this update. Now FB is opening without any issues.
